playbooks/app_install/roles/codesign_install/tasks/main.yml

```diff
@@ -24,3 +24,7 @@
     repo: 'https://wwwin-github.cisco.com/STO-Image-Signing/signature-file-verification.git'
     dest: '/home/jenkins/verification_scripts'
     version: 'master'
+ - name: Install cosgin version 2.2.2 rpm
+      yum:
+        name: ' https://engci-maven.cisco.com/artifactory/cmsp-sapphire-development-group/up-sac-utils/jenkins/v1/cosign-2.2.2-1.aarch64.rpm'
```

Comment:
Probably doesn't matter but remove the leading whitespace from name

Reply:
deleted